src/card.css

```diff
@@ -0,0 +1,91 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+
+  width: 200px;
+  padding: 16px;
+
+  border: 1px solid var(--card-border-color);
+  border-radius: 5px;
+}
+
+.card__image {
+  padding: 16px 3px 40px;
+  height: 134px;
+}
+
+.card__product-name {
+  padding-bottom: 4px;
```

Comment:
Should be margin-bottom instead, because it's not an inner area of an element. It sets a distance between two elements. 
Apply for other cases as well

Reply:
done